src/app/mapControls/FiltersContainer.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'app/config',
+    'app/mapControls/Filter',
+
+    'dijit/_TemplatedMixin',
+    'dijit/_WidgetBase',
+    'dijit/_WidgetsInTemplateMixin',
+
+    'dojo/_base/declare',
+    'dojo/_base/lang',
+    'dojo/dom-construct',
+    'dojo/text!app/mapControls/templates/FiltersContainer.html',
+    'dojo/topic'
+], function (
+    config,
+    Filter,
+
+    _TemplatedMixin,
+    _WidgetBase,
+    _WidgetsInTemplateMixin,
+
+    declare,
+    lang,
+    domConstruct,
+    template,
+    topic
+) {
+    return declare([_WidgetBase, _TemplatedMixin, _WidgetsInTemplateMixin], {
+        // description:
+        //      Holds filter widgets and coordinates their work
+        templateString: template,
+        baseClass: 'filters-container',
+        widgetsInTemplate: true,
+
+        // filters: _Filter[]
+        //      list of filters for this widget
+        filters: null,
+
+        // Properties to be sent into constructor
+
+        postCreate: function () {
+            // summary:
+            //      Overrides method of same name in dijit._Widget.
+            console.log('app.mapControls.FiltersContainer::postCreate', arguments);
+
+            this.filters = [
+                new Filter({
+                    name: 'Project Status',
+                    items: config.lut.projectStatus.map(function (item) {
+                        return [item, item];
+                    }),
+                    fieldName: config.fieldNames.Status,
+                    fieldType: Filter.TYPE_TEXT
+                }, domConstruct.create('div', null, this.container)),
+                new Filter({
+                    name: 'Feature Type',
+                    items: config.lut.featureType,
+                    fieldName: config.fieldNames.TypeCode,
+                    fieldType: Filter.TYPE_NUMBER,
+                    relatedTableQuery: true,
+                    anyAllToggle: true
+                }, domConstruct.create('div', null, this.container))
+            ];
+            this.filters.forEach(function (f) {
```

Comment:
I think that it reads better as part of the loop. Also, I'm not sure that it saves event handlers since I'm assuming that on creates one for each element that matches the expression. I say that we leave it.